caluma/caluma_form/tests/test_document.py

```diff
@@ -865,7 +866,7 @@ def test_save_document_answer(  # noqa:C901
             )
             snapshot.assert_match(answer)
         else:
-            with pytest.raises(Exception):
+            with pytest.raises((CustomValidationError, ValidationError)):
```

Comment:
Same here, this is a redundancy: `issubclass(CustomValidationError, ValidationError) == True`

Reply:
done